plugins/inputs/fluentd/fluentd.go

```diff
@@ -0,0 +1,194 @@
+package fluentd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"reflect"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const (
+	measurement  = "fluentd"
+	description  = "Read metrics exposed by fluentd in_monitor plugin"
+	sampleConfig = `
+	## This plugin only reads information exposed by fluentd using /api/plugins.json.
+	##
+	## Endpoint: 
+	## - only one URI is allowed
+	## - https is not supported
+	endpoint = "http://localhost:24220/api/plugins.json"
+	
+	## Define which plugins have to be excluded (based on "type" field - e.g. monitor_agent)
+	exclude = [
+		"monitor_agent",
+		"dummy",
+	]
```

Comment:
Make sure the sample config has 2 space indention here and in the README, so that it will have the same indention as with the other plugins when you run `telegraf config`.